apps/communications/contacts/js/contacts.js

```diff
@@ -415,7 +405,11 @@ var Contacts = (function() {
       ActivityHandler.postCancel();
       MainNavigation.home();
     } else {
-      handleBack();
+      handleBack(function() {
+        if (MainNavigation.currentView() === 'view-contact-details') {
+          contactsDetails.startNFC(currentContact);
```

Comment:
Ok, to have this right now, but once we have details completely separated in it's own html, we can use the visibility api to stop/start nfc.